docker/base/base.inc

```diff
@@ -1,6 +1,12 @@
 listen 8080;
 root /usr/wikia/slot1/current/src;
 
+set $fastcgi_server_name $host;
+
+if ($http_x_original_host) {
+	set $fastcgi_server_name $http_x_original_host;
```

Comment:
does it mean we should to Vary the response on X-Original-Host?

Reply:
I don't think we need to make such a change, as the part inside the if block is the current behavior. All MW requests coming into the cluster have `X-Original-Host` header set already.

Reply:
👍